src/test/java/com/puppycrawl/tools/checkstyle/AstTreeStringPrinterTest.java

```diff
@@ -70,6 +71,17 @@ public void testParseFile() throws Exception {
     }
 
     @Test
+    public void testPrintAst() throws Exception {
```

Comment:
This test looks oddly similar to `testParseFile` except for `getAbsoluteFile` and system's `file.encoding`, yet the test name doesn't describe this any by `testPrintAst`. Other test is named `testParseFile`.

Can we name this test something more distinct to describe what it is doing and how it is different than the other test?